python-checks/src/test/resources/checks/regex/possessiveQuantifierContinuationCheck.py

```diff
@@ -1,7 +1,27 @@
 import re
 
-match1 = re.match(r"a++abc", "aaaabc", re.DOTALL) # Noncompliant {{Change this impossible to match sub-pattern that conflicts with the previous possessive quantifier.}}
-match2 = re.match(r"\d*+[02468]", "1234", re.DOTALL) # Noncompliant {{Change this impossible to match sub-pattern that conflicts with the previous possessive quantifier.}}
+def foo():
+    match1 = re.match(r"a++abc", "aaaabc", re.DOTALL) # Noncompliant {{Change this impossible to match sub-pattern that conflicts with the previous possessive quantifier.}}
```

Comment:
Could you also add an assertion on issue location for one of the issues?
In general, it's important that we have at least one assertion on the issue message and one for the primary (and secondary when applicable) location for each rule test.